app/models/JobDescription.js

```diff
@@ -0,0 +1,43 @@
+const mongoose = require('mongoose');
+const CONSTANTS = require('../../config/constants');
+
+const Schema = mongoose.Schema;
+const jobDescriptionSchema = new Schema(
+    {
+        profile: {
+            type: String,
+            required: true,
+        },
+        organization: {
+            type: String,
+            required: true,
+        },
+        location: {
+            type: String,
+            required: true,
+        },
+        start_date: {
+            type: String,
+            required: true,
+        },
+        currently_working: {
+            type: Boolean,
+        },
+        end_date: {
+            type: String,
```

Comment:
This should be of `date` type.

Reply:
interviewee only needs to enter the month from when he/she has started working for a job or switched from a job.
if `Date` type is set then user needs to enter specific date to be saved in database which might look difficult for many of them to enter.